### PR TITLE
Snakemake: Add tiled_add kernel

### PR DESCRIPTION
**Pull request by JosseVanDelm**
Awaits: #317 

Note that I explicitly disable the benchmark test, since this is will be replaced by snakemake soon anyway

### CODE REVIEW

File: kernels/tiled_add/Snakefile [hunk 1]
```diff
@@ -0,0 +1,149 @@
+from util.snake.configs import get_snax_alu_config
+
+config = get_snax_alu_config()
+
+# Untiled example needs preprocessing:
+# FIXME: Does it really need preprocessing though?
```
Comment (jorendumoulin) [inline kernels/tiled_add/Snakefile:6]: Fix? If it doesn.t, just call it .preprocfinal.mlir?

File: kernels/tiled_add/Snakefile [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+
+
+module default_rules:
+    snakefile:
+        "../../util/snake/default_rules.smk"
+    config:
+        config
+
+
+use rule * from default_rules exclude snax_compile_mlir as default_*
```
Comment (jorendumoulin) [inline kernels/tiled_add/Snakefile:59]: again the exclude

**Review by jorendumoulin — APPROVED** [review]

File: kernels/tiled_add/Snakefile (hunk 2 above)
Comment (JosseVanDelm) [reply kernels/tiled_add/Snakefile]: Done!

File: kernels/tiled_add/Snakefile (hunk 1 above)
Comment (JosseVanDelm) [reply kernels/tiled_add/Snakefile]: Turns out it still needed it!

**Pull request merged**